Mathlib/AlgebraicGeometry/AffineScheme.lean

```diff
@@ -787,6 +787,7 @@ theorem self_le_basicOpen_union_iff (s : Set Γ(X, U)) :
 
 end IsAffineOpen
 
+set_option maxHeartbeats 400000 in
```

Comment:
This proof needs heavy cleanup, specifically it seems we should get rid of the second `show` statement, but I couldn't figure out how to get that done...

Reply:
It's still slower than before, but now it's doable. So I think we should be OK for getting this merged.